### ISSUE TITLE
More of a feature request: switchable associations

### ISSUE THREAD

**Issue by wulftone** [notahat/machinist#100]
I've found myself frequently creating two blueprints for a single model, one with associations, one without.  It would be great if there was a simple switch to turn off/on the creation of related records...

**Comment by scottkf** [comment]
+1 for this. Is this project even still active though?

**Comment by rngtng** [comment]
I didn't test, but [according to wiki](https://github.com/notahat/machinist/wiki/Make) this is possible by passing `nil` for the association.

@scottkf I'm wondering too if project is still active, very sad if it wouldn't :-( - @notahat ?

**Comment by wulftone** [comment]
True, but I was thinking of a more succinct way, like `User.make!(:no_relations)` or something (which is what I currently do, but I have to explicitly type out a new blueprint for this).  Some models have lots of associations, so it'd start to be a pain to have to turn all of them off.  I might add this functionality myself once my current project is over.
